binderhub-service/templates/_helpers-labels.tpl

```diff
@@ -0,0 +1,19 @@
+{{- /*
+  Common labels
+*/}}
+{{- define "binderhub-service.labels" -}}
+helm.sh/chart: {{ printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{ include "binderhub-service.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{- /*
+  Selector labels
+*/}}
+{{- define "binderhub-service.selectorLabels" -}}
+app.kubernetes.io/name: {{ .Values.nameOverride | default .Chart.Name | trunc 63 | trimSuffix "-" }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```

Comment:
I relocated these to a dedicated file to keep track of what helper functions was related to: labels, names, and other misc things.